Libs/MRML/Widgets/qMRMLSceneCategoryModel.cxx

```diff
@@ -85,7 +88,7 @@ QStandardItem* qMRMLSceneCategoryModel::itemFromCategory(const QString& category
 //------------------------------------------------------------------------------
 int qMRMLSceneCategoryModel::categoryCount()const
 {
-  return this->match(this->mrmlSceneIndex().child(0,0),
+  return this->match(ctk::modelChildIndex(const_cast<qMRMLSceneCategoryModel*>(this), this->mrmlSceneIndex(), 0, 0),
```

Comment:
After https://github.com/commontk/CTK/pull/1014 is integrated, this could be simplified.

Reply:
For reference, simplified version  of this PR has been pushed here: https://github.com/jcfr/Slicer/commits/update-ctk-with-pr-1014